src/Sfx-Standalone/.build/tasks/pack.licensing.js

```diff
@@ -283,6 +283,7 @@ function generateThirdPartyNotice(deps, noticeFilePath) {
         fs.appendFileSync(noticeFd, "This project incorporates components from the projects listed below. The original copyright notices and the licenses under which Microsoft received such components are set forth below. Microsoft reserves all rights not expressly granted herein, whether by implication, estoppel or otherwise.\r\n");
         fs.appendFileSync(noticeFd, "\r\n");
 
+        deps = deps.filter((value, index, self) => self.findIndex((item) => item.name === value.name) === index);

```

Comment:
Seems the self and deps share the same index order. so why it's necessary to do a findIndex in the self?

Reply:
This is to remove duplication. Otherwise there will be two angulars and two jquerys.

Reply:
Set can be used here to reduce the complexity and improve the performance. :)